### ISSUE TITLE
tensorflow to tensorrt

### ISSUE THREAD

**Issue by Hao-Zhao** [parallel-forall/code-samples#8]
Hi, 

I am trying to convert tensorflow model into tensorrt, I did import successfully, but the convolution layer output is completely different, I guess that is due to the difference of order in convolutional layer weights in tf(RSCK ) and trt(KCRS). 

Did you found same problem before?

Thanks

**Comment by faustomilletari** [comment]
Hello, that should be taken into account automatically when the conversion happens. Did you try to run the example with the flowers dataset?

**Comment by Hao-Zhao** [comment]
Hi, 
Thanks for your reply. I have not tried the example yet. The one I imported from tensorflow is by tensorflow slim interface, and tensorrt does not support slim yet, so I guess that's the problem. Only can solve the problem if I know the difference between slim and normal tensorflow.

**Comment by faustomilletari** [comment]
Hello,

We have used slim for a few things but not everything is supported yet. For example batch norm is an exception. My advice is to get started with either keras or pure tf and then go into more  complex things once the basics are in place. 

Good luck,

Fausto Milletarì

**Comment by Hao-Zhao** [comment]
Hi Fausto,

You are right, thanks again. When I figure out, I will tell you.

Thanks
Hao
